create-new-release.sh

```diff
@@ -29,6 +29,9 @@ echo "OLD_SHORT_VERSION = $OLD_SHORT_VERSION"
 
```

Comment:
@wraschke Just noticed the file rename. The reference should be updated: https://github.com/OpenLiberty/ci.docker/blob/95fbbf9b2ed47e51180d10efac425988e42bce1f/.github/workflows/update_files_for_release.yml#L48